### ISSUE TITLE
Grafana dashboard source

### ISSUE THREAD

**Issue by Impatience71** [retzkek/chiamon#34]
Hi all, 

its more a question then an issue: 
the grafana dashboard in the source tree looks a bit different from the one which is copied in the readme file. Would it be possible to publish also the dashboard from the readme ? (I think mainly the disk temperature is missing).
Thanks in advance !
Its a really beautiful and comprehensive dashboard ! Thanks once more for this great work !
Cheers Peter

**Comment by retzkek** [comment]
Hi Peter,
My personal dashboard (what I took the screenshot of) is rather customized for my setup, and has actually changed quite a bit as I have moved from plotting to just farming. The power of Grafana is how simple it is to change dashboards, so I really recommend diving in and tailoring it to your needs. 

Adding disk temperatures requires another metric collector (actually a sort of [plugin to node_exporter](https://github.com/prometheus-community/node-exporter-textfile-collector-scripts/blob/master/smartmon.py)), which just adds more complexity to the setup, which is already causing beginners a lot of problems. Although I suppose it's worth pointing out, for people on Ubuntu >20.04 at least, that `apt install prometheus` or `apt install prometheus-node-exporter` will by default include the SMART monitoring.
